src/client/components/ProfileFormComponent/ProfileFormComponent.css

```diff
@@ -0,0 +1,73 @@
+@import url('https://fonts.googleapis.com/css2?family=Roboto+Mono:wght@400;600&family=Roboto:wght@300&display=swap');
+
+.avatar-Image img {
+  padding: 10px 40px;
+  display: flex;
+  margin: 40px auto;
+  justify-content: center;
+  width: 280px;
```

Comment:
I guess, we can use some % instead of px value to width so that it doesn't break :) Tried the following, please check.
`width: 60%;
  max-width: 250px;`